src/twoband.jl

```diff
@@ -30,16 +30,14 @@ function sum(w::TwoBandWeight{T}) where T
     if 2w.a == 2w.b == -1 && 2w.c == 1
         convert(T,π)
     else
-        error("not implemented")
-#         1/2 \[Pi] (-((\[Rho]^(1 + 2 b + 2 c)
-#       Gamma[1 + b] Hypergeometric2F1Regularized[-a, 1/2 + c,
-#       3/2 + b + c, \[Rho]^2])/Gamma[1/2 - c]) + (
-#    Gamma[1 + a] Hypergeometric2F1Regularized[-b, -(1/2) - a - b - c,
-#      1/2 - b - c, \[Rho]^2])/
-#    Gamma[3/2 + a + b + c]) Sec[(b + c) \[Pi]]
+        # error("not implemented.")
+        a = w.a; b = w.b; c = w.c; ρ = w.ρ
+        π/2 * (-((ρ^(1 + 2b + 2c) * gamma(1 + b) * _₂F₁(-a, 1/2 + c, 3/2 + b + c, ρ^2))/gamma(1/2 - c)) 
```

Comment:
Needs to be type-stable:
```suggestion
        convert(T,π)/2 * (-((ρ^(1 + 2b + 2c) * gamma(1 + b) * _₂F₁(-a, one(T)/2 + c, convert(T,3)/2 + b + c, ρ^2))/gamma(one(T)/2 - c)) +
```